api/repository/pokemons.repo.ts

```diff
@@ -0,0 +1,30 @@
+import { Prisma } from "@prisma/client";
+import { Pokemon } from "../interfaces/pokemons.interfaces";
+import { prisma } from "../utils/prisma";
+
+export const pokemonRepo = {
+  getUserPokemons,
+  getRandomPokemon,
+};
+
+async function getUserPokemons(userId: string) {
+  const ownedPokemons = await prisma.userPokemon.findMany({
+    where: {
+      userId: userId,
+    },
+    select: {
+      pokemonId: true,
+    },
+  });
+
+  return ownedPokemons.map((p) => p.pokemonId);
```

Comment:
implement with query raw and return from the query the desired fields instead of fetching and then mapping